flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/scala/expressionDsl.scala

```diff
@@ -544,11 +544,22 @@ trait ImplicitExpressionOperations {
   def overlay(newString: Expression, starting: Expression, length: Expression) =
     Overlay(expr, newString, starting, length)
 
+  /**
+    * Returns the numeric value of the leftmost character of the string.
+    */
+  def ascii() = Ascii(expr)
+
   /**
     * Returns the base string decoded with base64.
     */
   def fromBase64() = FromBase64(expr)
 
+  /*
+    * Returns string contains a character which converts from a ASCII integer.
```

Comment:
Returns the character corresponding to the input ASCII integer. If the ASCII code less then 0 or greater than 255, returns null.